pom.xml

```diff
@@ -646,11 +650,92 @@
                 <version>${thrift.version}</version>
                 <scope>compile</scope>
             </dependency>
+			<!-- used by examples/storm-starter -->
             <dependency>
                 <groupId>junit</groupId>
-                <artifactId>junit</artifactId>
+		        <artifactId>junit</artifactId>
                 <version>${junit.version}</version>
+		        <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.hadoop</groupId>
+                <artifactId>hadoop-client</artifactId>
+                <version>${hadoop.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>log4j</groupId>
+                        <artifactId>log4j</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.slf4j</groupId>
+                        <artifactId>slf4j-api</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.slf4j</groupId>
+                        <artifactId>slf4j-log4j12</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
```

Comment:
Concerned about dependency conflicts.  Can we have the HDFS-specific deps pulled in from external HDFS instead?